common/src/main/java/com/wynntils/services/mapdata/attributes/type/MapAttributes.java

```diff
@@ -20,7 +20,10 @@ public interface MapAttributes {
     // 0 means no value specified; inherit
     int getPriority();
 
-    // 0 means no information is available, or suitable for all combat levels
+    // the minimum combat level for which this feature is suitable for
+    // 1 means suitable for all levels (players start at level 1)
+    // 0 means inherit
+    // -1 means no information is available, or the feature is suitable for all levels
```

Comment:
So can we set -1 as the combat level at the "root"? So if someone adds a data point and leaves out the level, the default value will be "no info".

Reply:
Every other method here specified their default as inherit. I think this method should do the same, and we'll specify the real default value somewhere else (which will be called when the root is "inherit".

Reply:
Ok. I've apparently forgotten like everything about this. :) Where do we specify the root? Can we make sure this is set to -1?

Reply:
I am a bit lost in that regard still. Here is what I've found. It seems that `FullFeatureAttributes` and `FullCategoryAttributes` expect inherit values to be null, or 0. I would assume these classes would provide/get the default values. Somewhere, sometime. I think i'll have to figure this out, when I am working on porting to map data.

(For reference, here is a code snippet from `FullFeatureAttributes`:
```java
    protected <T> T getAttribute(Function<MapAttributes, T> getter) {
        // Check if the feature has overridden this attribute
        if (attributes != null) {
            T attribute = getter.apply(attributes);
            if (attribute != null && !(attribute instanceof Integer i && i == 0)) {
                return attribute;
            }
        }

        // Otherwise get it from the category
        MapAttributes categoryAttributes = Services.MapData.getFullCategoryAttributes(feature.getCategoryId());
        if (categoryAttributes == null) return null;

        return getter.apply(categoryAttributes);
    }
```

Reply:
This is the one point that is still worrying me. I think we need to get to the bottom on how this is supposed to work, so we get this part right.

Maybe it is bad that we use 0 for inheritance on the other integer values as well. Perhaps we need a way to say if an integer value is present or not, basically having nullable `Integer`s instead of `int`. That is harder to match to a json format, though.

Reply:
So, do you have any idea on how to handle this? Nullable ints are not so bad to handle in GSON, afaik.